main.py

```diff
@@ -148,6 +219,15 @@ def setSaveFileName(self, obj):
 		if fileName:
 			obj.setText(fileName)
 
+	def setDirectory(self, obj):
+		options = QFileDialog.Options()
+		directory = QFileDialog.getExistingDirectory(self,
+				"Select DVD Directory",
```

Comment:
nit: not only "DVD", but "Optical Disc Image", poor wording though. I'd probably pick "CD / DVD Image" or something.